content/es/community/_index.html

```diff
@@ -0,0 +1,62 @@
+---
+title: Comunidad
+layout: basic
+cid: community
+---
+
+<section id="mainContent">
+    <main>
+        <div class="content">
+            <h3>Garantizando el funcionamiento de Kubernetes para todo el mundo y en cualquier lugar.</h3>
+            <p>Conecte con la comunidad Kubernetes en nuestro canal de <a href="http://slack.k8s.io/">Slack</a>, <a href="https://discuss.kubernetes.io/">foro</a>, o únete al
+                <a href="https://groups.google.com/forum/#!forum/kubernetes-dev">Kubernetes-dev Google group</a>.
+                Cada semana se lleva a cabo una reunión de la comunidad  por videoconferencia para discutir el estado de cosas, revise el documento
+                <a href="https://github.com/kubernetes/community/blob/master/events/community-meeting.md "> Community Meeting </a> para obtener información sobre cómo participar. </p>
+            <p>También puede formar parte de la comunidad en cualquier parte del mundo a través de la
+                <a href="https://www.meetup.com/topics/kubernetes/">Comunidad de Meetups sobre Kubernets </a>o el
```

Comment:
s/Kubernets/Kubernetes

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/e0ef1a231dd9935e8647feff4c4b04f0b32e4010